lib/data-types.d.ts

```diff
@@ -396,105 +512,128 @@ export class VIRTUAL<T> extends ABSTRACT {
  *
  * @property ENUM
  */
-export class ENUM<T extends string> extends ABSTRACT {
-  constructor(...values: string[]);
-  constructor(options: ENUM_Options<T>);
-  values: T[];
-  options: ENUM_Options<T>;
+export const ENUM: EnumDataTypeStatic;
+
+export interface EnumDataTypeStatic extends AbstractDataTypeStatic {
```

Comment:
Alright, didn't think about that usecase. I always used a pattern like this to have string enums in TS:
```ts
import {values} from 'lodash';
type UserRole = 'admin' | 'normal';
const UserRole = {
  ADMIN: 'admin' as 'admin',
  NORMAL: 'normal' as 'normal'
}
interface IUserAttributes {
  role: UserRole;
}
interface IUser extends IUserAttributes, Sequelize.Instance<IUser, IAttributes> {}
const User = sequelize.define<IUser, IUserAttributes>('User', {
  role: Sequelize.ENUM(...values(UserRole) as string[])
}
User.build({role: UserRole.ADMIN});